### DISCUSSION TITLE
Defining a variable

### DISCUSSION THREAD

**Discussion by mcnuggetsx20** [hyprwm/Hyprland#6733] [Q&A] [answered]
I want to run dmenu with the following command:
dmenu_run -sb '#D0D0D0' -nf '#f0af16' -sf '#000000' -nb '#1d1b1b' -c -l 30
but I have no idea how to put it inside a variable and bind it to a key. No matter what I do to escape the '#' symbols it doesn't work

**Comment by fufexan** [comment, marked as answer]
https://wiki.hyprland.org/Configuring/Configuring-Hyprland/#comments
tl;dr: use `##` to escape pound signs.
